### PR TITLE
[C-458] Fix CollectiblesTab text and links

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Fixes various text and link issues
- Fixes `Collectibles by ${artist}`
- Fixes share button
- Fixes collectible details drawer share button url

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-458-fix-collectibles

File: packages/mobile/src/utils/routes.tsx [hunk 1]
```diff
@@ -82,3 +82,20 @@ export const getEmptyPageRoute = (fullUrl = false) => {
 export const getAudioPageRoute = () => {
   return '/audio'
 }
+
+/**
+ * Generate a short base36 hash for a given string.
+ * Used to generate short hashes for for queries and urls.
```
Comment (sliptype) [inline packages/mobile/src/utils/routes.tsx:88]:
```suggestion
 * Used to generate short hashes for queries and urls.
```

**Review by sliptype — APPROVED** [review]
Amazing, thank you!

**Review by Kyle-Shanks — APPROVED** [review]
rad

**Pull request merged**